test/e2e/storage/testsuites/subpath.go

```diff
@@ -619,12 +636,19 @@ func waitForPodSubpathError(f *framework.Framework, pod *v1.Pod) error {
 				case status.State.Running != nil:
 					return false, fmt.Errorf("subpath container unexpectedly became running")
 				case status.State.Terminated != nil:
+					if status.State.Terminated.ExitCode != 0 {
+						return true, nil
+					}
 					return false, fmt.Errorf("subpath container unexpectedly terminated")
 				case status.State.Waiting != nil:
 					if status.State.Waiting.Reason == "CreateContainerConfigError" &&
 						strings.Contains(status.State.Waiting.Message, "subPath") {
 						return true, nil
 					}
+					if status.State.Waiting.Reason == "CreateContainerConfigError" &&
```

Comment:
Do we still need this?

Reply:
I don't think so. I haven't seen this happening during testing